packages/yoastseo/spec/scoring/assessments/seo/KeyphraseInImageTextAssessmentSpec.js

```diff
@@ -1,22 +1,77 @@
+			"<a href='https://yoa.st/4f6' target='_blank'>Add some images with alt attributes that include the keyphrase or synonyms</a>!" );
+	} );
+
+	it( "should show feedback for a page without a text and keyphrase", function() {
+		const paper = new Paper( "", { keyword: "" } );
+		const researcher = new EnglishResearcher( paper );
+		buildTree( paper, researcher );
+
+		const assessment = new KeyphraseInImagesAssessment().getResult( paper, researcher );
+
+		expect( assessment.getScore() ).toEqual( 3 );
+		expect( assessment.getText() ).toEqual( "<a href='https://yoa.st/4f7' target='_blank'>Image Keyphrase</a>: This page does not have images, a keyphrase, or both. " +
+			"<a href='https://yoa.st/4f6' target='_blank'>Add some images with alt attributes that include the keyphrase or synonyms</a>!" );
+	} );
+
+	it( "should show feedback for a page with a text and without a keyphrase", function() {
+		const paper = new Paper( "These are just five words.", { keyword: "" } );
+		const researcher = new EnglishResearcher( paper );
+		buildTree( paper, researcher );
+
+		const assessment = new KeyphraseInImagesAssessment().getResult( paper, researcher );
+
+		expect( assessment.getScore() ).toEqual( 3 );
+		expect( assessment.getText() ).toEqual( "<a href='https://yoa.st/4f7' target='_blank'>Image Keyphrase</a>: This page does not have images, a keyphrase, or both. " +
+			"<a href='https://yoa.st/4f6' target='_blank'>Add some images with alt attributes that include the keyphrase or synonyms</a>!" );
+	} );
+
+	it( "should show feedback for a page with a keyphrase and without a text", function() {
+		const paper = new Paper( "", { keyword: "keyword" } );
+		const researcher = new EnglishResearcher( paper );
+		buildTree( paper, researcher );
+
+		const assessment = new KeyphraseInImagesAssessment().getResult( paper, researcher );
+
+		expect( assessment.getScore() ).toEqual( 3 );
+		expect( assessment.getText() ).toEqual( "<a href='https://yoa.st/4f7' target='_blank'>Image Keyphrase</a>: This page does not have images, a keyphrase, or both. " +
+			"<a href='https://yoa.st/4f6' target='_blank'>Add some images with alt attributes that include the keyphrase or synonyms</a>!" );
+	} );
+
+	it( "should show feedback for a page without a keyphrase and with an image without alt tag", function() {
+		const paper = new Paper( "These are just five words <img src='image.jpg' alt='' />", { keyword: "" } );
+		const researcher = new EnglishResearcher( paper );
+		buildTree( paper, researcher );
+
+		const assessment = new KeyphraseInImagesAssessment().getResult( paper, researcher );
+
+		expect( assessment.getScore() ).toEqual( 6 );
+		expect( assessment.getText() ).toEqual( "<a href='https://yoa.st/4f7' target='_blank'>Image Keyphrase</a>: Images on this page do not have alt attributes that reflect the topic of your text. <a href='https://yoa.st/4f6' target='_blank'>Add your keyphrase or synonyms to the alt tags of relevant images</a>!" );
+	} );
```

Comment:
Is this the desired behavior? I thought that we should show `Image Keyphrase: This page does not have images, a keyphrase, or both.` in this case.